en/SUMMARY.md

```diff
@@ -25,10 +25,10 @@
 * [Simulation](simulation/README.md)
   * [jMAVSim Simulation](simulation/jmavsim.md)
   * [Gazebo Simulation](simulation/gazebo.md)
-  * [HITL Simulation](simulation/hitl.md)
-  * [Interfacing to ROS](simulation/ros_interface.md)
   * [AirSim Simulation](simulation/airsim.md)
   * [Multi-Vehicle Simulation](simulation/multi-vehicle-simulation.md)
+  * [HITL Simulation](simulation/hitl.md)
+  * [Interfacing to ROS](simulation/ros_interface.md)
```

Comment:
@TSC21 I would like to move this OUT of simulation and into the ROS section. My reasoning is that even though ROS can be used with a simulator, it can also be used stand alone. The topic name would also change to something like "ROS with Gazebo Simulation". Note that I have updated the Gazebo doc to better link to this ROS doc too.

Does this make sense to you?

Reply:
Makes sense to me

Reply:
OK, done in https://github.com/PX4/Devguide/pull/393/commits/108d2cea39e7c4e6ca20729211c20fe4e48e6179